docs/supportfeedback.md

```diff
@@ -5,17 +5,18 @@ hide_title: true
 sidebar_label: Support, Feedback, & Updates
 ---
 
-# ![Banner Support, Feedback,   Updates](https://github.com/user-attachments/assets/5e5b2ad8-8efa-4950-baa4-d315a3f252cf)
+# ![Banner Support, Feedback, & Updates](https://github.com/user-attachments/assets/5e5b2ad8-8efa-4950-baa4-d315a3f252cf)
 
 :::warning When Seeking Support
-***DO NOT*** ask for support in the Tale of Two Wastelands, Wasteland Supplemental, ModdingLinked, non-Uranium Fever channels in Sweets Retreat, or any other unaffiliated discord channel.
+***DO NOT*** ask for support in the Tale of Two Wastelands, Wasteland Supplemental, ModdingLinked, non-Uranium Fever channels in Sweets Retreat, or any other unaffiliated Discord channel.
 
 ***DO NOT*** ping **OR** private message any member of the Uranium Fever team for support or questions. ***DO NOT*** ask non-support members for support.
+:::
 
 :::danger Altering Uranium Fever
 Altered Uranium Fever lists ***WILL NOT*** be supported.
 :::
 
 - Post your support requests and feedback in the **APPROPRIATE** **Uranium Fever channels** in [Sweets Retreat](https://discord.gg/Uu6gZZSaeA).
 
-	![Discord](https://github.com/user-attachments/assets/e13e3d65-419b-4294-831d-5bbcbcea9e64)
```

Comment:
Please don't change the format of the indentations, it makes it harder for me to manage when I use tab to indent. Once this change is removed I'll merge this one.